client/server/render/index.js

```diff
@@ -122,24 +122,13 @@ export function render( element, key = JSON.stringify( element ), req ) {
 
 		return renderedLayout;
 	} catch ( ex ) {
-		if ( process.env.NODE_ENV === 'development' ) {
-			throw ex;
-		} else {
-			try {
-				req.context.store.dispatch(
-					logToLogstash( {
-						feature: 'calypso_ssr',
-						message: 'Exception thrown on render',
-						user_id: req.context.user?.ID,
-						extra: {
-							message: ex.message,
-							stack: ex.stack,
-						},
-					} )
-				);
-			} catch {
-				// Failed to log the error, swallow it so it doesn't break anything. This will serve
-				// a blank page and the client will render on top of it.
+		try {
+			req.logger.error( ex );
```

Comment:
Are we now also logging the errors for dev env? Or does bunyan intentionally ignore the dev environment?

Reply:
Bunyan will log the error in all modes, including dev. The only difference in local development is that we use [bunyan-cli](http://trentm.com/node-bunyan/bunyan.1.html) to prettify the logs

Reply:
How useful are logged errors from the `development` environment? I'd suggest disabling logging for it, just like we didn't log before. Or do you have a good reason to log the dev environment?

Reply:
(to make sure we are on the same page: this is logging them to the console).

What are the alternative, hide them? I surely don't want to hide errors in dev mode, how would the dev fix them?

Reply:
Right, I see! Makes sense. We definitely don't want to hide any errors from appearing in the console.

I see the confusion here 😉  We're talking about two different things. You're talking about logging in the console and I'm talking about logging in logstash.

So, previously, we were using `logToLogstash()` which was sending data to logstash over the REST API, and now we're removing that in favor of the bunyan logs. While it's useful they'll appear in the console, should we keep sending those errors to logstash for all-but-dev environments?

Reply:
They are already in Logstash. When running in development `bunyan` will write logs to the console, but when running in production it will write logs to the filesystem. Then a process built by Systems will read those logs and send them to logstash (more info in pMz3w-c0R-p2)

Reply:
This is exactly what I was asking in the first place 👍  Nice! Thanks for elaborating